package.json

```diff
@@ -32,11 +32,11 @@
   "devDependencies": {
     "@types/body-parser": "0.0.33",
     "@types/chai": "^3.4.34",
-    "@types/cheerio": "^0.17.31",
```

Comment:
If `cheerio` is still needed, this is obviously still needed, just a reminder comment.